### DISCUSSION TITLE
Printing environment variable through OS returns none on printing.

### DISCUSSION THREAD

**Discussion by Noveleader** [smartcontractkit/full-blockchain-solidity-course-py#1377] [Q&A] [answered]
This is my code and I set the new environment variable named ```PRIVATE_KEY``` and imported OS and tried accessing my private key but it returns none when I try to print my ```private key``` . I am on ```WINDOWS``` .
![image](https://user-images.githubusercontent.com/91677627/162141015-527c5344-3795-4262-b438-fcd593092cb4.png)

**Comment by Noveleader** [comment, marked as answer]
I was ignoring the github I did my research and [this](https://github.com/smartcontractkit/full-blockchain-solidity-course-py#interacting-with-our-contract-in-python--web3py) should help.

**Reply by cromewar** [reply]
I'm glad you figured out by yourself.

**Reply by Noveleader** [reply]
yeah
